src/elementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -6,15 +6,14 @@ const ApplicationViewWrapper: React.FC<ApplicationViewProps> = (props) => {
   const {
     templateElement: { elementTypePluginCode: pluginCode },
     isVisible,
-    isEditable,
-    isRequired,
```

Comment:
was wondering if isRequired would be something that ApplicationViewWrapper would display (generic for all plugins)

Reply:
Yeah, fair enough. Have put them back in here. 762aa63